### DISCUSSION TITLE
OpenId public signing key support

### DISCUSSION THREAD

**Discussion by LoupLabelle** [Leantime/leantime#2010] [Q&A] [answered]
I've configured leantime with authelia and I get this message when I use oidc to connect:
`The format for the public signing key is not yet supported, check if your provider supports other jwks endpoints.`

Are there plans to support this kind of public signing oidc?

**Comment by LarssonOliver** [comment]
I stumbled upon the same error, also using Authelia.

**Comment by gloriafolaron** [comment, marked as answer]
Thanks for sharing this! 

That one hasn't been on the radar at the moment but definitely something we'd be happy to look at in the coming future. 

In transparency, and at the moment, updates like this may be a bit out on our roadmap unless sponsored as a feature or community contributed.  We're currently working to grow our resources so that we can get additional capacity.

**Comment by erikthegamer1242** [comment]
> I've configured leantime with authelia and I get this message when I use oidc to connect: `The format for the public signing key is not yet supported, check if your provider supports other jwks endpoints.`
> 
> Are there plans to support this kind of public signing oidc?

Hello, I am having the same issue, how did you solve it?

**Reply by LoupLabelle** [reply]
I configured Authelia with a self-signed certificate with help from the [#Generate a certificate issued by own CA of this forum](https://bbs.archlinux.org/viewtopic.php?pid=1776753#p1776753).
I then merged the appropriate certificate files to make the certificate chain and directed authelia to use this file.
This enabled the jwks endpoints the application wanted.
